libs/pmp-web/repository/ui-table-pr-statistic/src/lib/table-pr-statistic/table-pr-statistic.component.html

```diff
@@ -38,6 +51,16 @@
     </mat-cell>
   </ng-container>
 
+  <!-- Comments count Column -->
+  <ng-container matColumnDef="commentsCount">
+    <mat-header-cell *matHeaderCellDef class="comments-count-column" mat-sort-header="commentsCount"
+      >Comments count
+    </mat-header-cell>
+    <mat-cell *matCellDef="let element" class="comments-count-column">
+      {{ element.commentsCount }}
```

Comment:
Please add comment icon. The same as on the /reviewer/id page
![image](https://user-images.githubusercontent.com/24508253/81160425-54d93000-8f8a-11ea-8a50-06e3cbe21ed5.png)